### DISCUSSION TITLE
Obfuscation WireGuard

### DISCUSSION THREAD

**Discussion by CookieSSH** [wg-easy/wg-easy#1252] [Q&A]
Hello. How can me combine WG and XRay Reality with each other, to that traffic was masked?

I tried to ask for help from the AI, but it offers me to set up Reality on the server and on the client, but I don't need the setting on the client. I want to send my friends only the WG configuration (qr-code) to connect and nothing else. How can these two programs be configured and connected?

I'm completely new to this, so I may not know many moments.

**Comment by tetuaoro** [comment]
Do you connect the 2 containers ? With docker network.

**Reply by CookieSSH** [reply]
> Do you connect the 2 containers ? With docker network.

Do you propose to use a container for Xray Reality and WG, then connect them together?

**Reply by tetuaoro** [reply]
> > Do you connect the 2 containers ? With docker network.
> 
> Do you propose to use a container for Xray Reality and WG, then connect them together?

Yes

**Reply by tetuaoro** [reply]
Or no, in fact, Xray just needs a configuration ip.

**Reply by CookieSSH** [reply]
> Or no, in fact, Xray just needs a configuration ip.

In what plan do you need an ip configuration for Xray? I just don't understand much about the Linux server part.
